composer.json

```diff
@@ -122,6 +122,7 @@
         "amphp/http-tunnel": "^1.0",
         "async-aws/ses": "^1.0",
         "async-aws/sqs": "^1.0",
+        "async-aws/sns": "^1.0",
```

Comment:
Why not `^1.2` like in the composer.json of the bridge?

Reply:
@OskarStark I update it to ^1.0 on composer.json of the bridge as he can work with 1.0.0 and upper versions.